src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1763,6 +1763,20 @@ bool isContainingAssemblyInReferences(ISymbol s)
         /// within this compilation, the reference is considered to be used. Etc.
         /// The returned set is a subset of references returned by <see cref="References"/> API.
         /// The result is undefined if the compilation contains errors.
+        /// 
+        /// The effect of imported namespaces on result of this API depends on whether reporting of
```

Comment:
Consider moving this to `<remarks>` section and using `<para>...</para>` to denote paragraphs (instead of blank lines).